src/Features/VisualBasic/Portable/Completion/CompletionProviders/AwaitCompletionProvider.vb

```diff
@@ -42,5 +44,52 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
         Private Protected Overrides Function GetAsyncSupportingDeclaration(token As SyntaxToken) As SyntaxNode
             Return token.GetAncestor(Function(node) node.IsAsyncSupportedFunctionSyntax())
         End Function
+
+        Private Protected Overrides Function GetTypeSymbolOfExpression(semanticModel As SemanticModel, potentialAwaitableExpression As SyntaxNode, cancellationToken As CancellationToken) As ITypeSymbol
```

Comment:
why are these private protected?

Reply:
That's what was already there, and I followed suit. Should I change it to just `protected`?

Reply:
yesplz

Reply:
Done in b5648160